content/200-orm/200-prisma-client/100-queries/035-select-fields.mdx

```diff
@@ -19,6 +19,8 @@ To customize the result:
 
 Selecting only the fields and relations that you require rather than relying on the default selection set can ✔ reduce the size of the response and ✔ improve query speed.
 
+When doing a relation query with `include` or by using `select` on a relation field, you can also specify the `relationLoadStrategy` to decide whether you want to use a database-level JOIN or perform multiple queries and merge the data on the application level. This feature is currently in [Preview](/orm/more/releases/preview), you can learn more about it [here](/orm/prisma-client/queries/relation-queries/relation-load-strategies-preview).
```

Comment:
```suggestion
Since version [5.9.0](https://github.com/prisma/prisma/releases/tag/5.9.0), when doing a relation query with `include` or by using `select` on a relation field, you can also specify the `relationLoadStrategy` to decide whether you want to use a database-level JOIN or perform multiple queries and merge the data on the application level. This feature is currently in [Preview](/orm/more/releases/preview), you can learn more about it [here](/orm/prisma-client/queries/relation-queries/relation-load-strategies-preview).
```